aiida/cmdline/commands/cmd_devel.py

```diff
@@ -48,6 +48,24 @@ def devel_check_load_time():
     echo.echo_success('no issues detected')
 
 
+@verdi_devel.command('check-undesired-imports')
+def devel_check_undesired_imports():
+    """Check that verdi does not import python modules it shouldn't.
+
+    Note: The blacklist was taken from the list of packages in the 'atomic_tools' extra but can be extended.
```

Comment:
Should we use this more dynamically instead by reading it in from `setup.json`, or is it preferable to keep it explicit and hard-coded here?

Reply:
Up for discussion of course. However, the setup.json handling code is currently under `utils/` i.e. not even in aiida-core.
Not sure it's worth the effort.